gatsby/content/blog/2022/09/2022-09-13-synapse-1.67.mdx

```diff
@@ -0,0 +1,72 @@
+---
+date: '2022-09-13'
+title: Synapse 1.67 released
+categories:
+- Releases
+author: Brendan Abolivier
+---
+
+
+It's that time again - Synapse release time! [Synapse
+1.67](https://github.com/matrix-org/synapse/releases/tag/v1.67.0) is fresh out
+of the oven, let's have a look at what's inside.
+
+## Removal of TCP replication
+
+As announced in the [Synapse 1.66 release
+announcement](https://matrix.org/blog/2022/09/02/synapse-1-66-released), Synapse 1.67 removes support for the legacy TCP replication protocol.
+
+As a reminder, Synapse can be configured to use workers to spread its load over
+multiple processes, or even multiple machines. These workers communicate between
+each other (and with the main Synapse process) using something we call
+"replication".
+
+[Synapse 1.18](https://github.com/matrix-org/synapse/releases/tag/v1.18.0)
+deprecated its TCP-based replication protocol, and replaced it with a brand new
+one using [Redis](https://redis.io/). As of Synapse 1.67, this Redis-based
+protocol is now the only protocol that can be used.
+
+See the [upgrade
+notes](https://matrix-org.github.io/synapse/v1.67/upgrade.html#direct-tcp-replication-is-no-longer-supported-migrate-to-redis)
+for more information.
+
+## Changes for installations using a source checkout
+
+Server admins who installed Synapse using a source checkout should be familiar
+with [Poetry](https://python-poetry.org/) by now. This is a tool we use to
```

Comment:
not entirely true; you can still use pip and forget about Poetry. (my homeserver still does this)

Reply:
Good point.